server/app.ts

```diff
@@ -46,7 +48,18 @@ export default function createApp(services: Services): express.Application {
   app.use(pdfRenderer(new GotenbergClient(config.apis.gotenberg.apiUrl)))
   app.use(authorisationMiddleware())
 
-  app.use('/', indexRoutes(standardRouter(services.userService), services))
+  if (process.env.DISPLAY_MAINTENANCE_PAGE === 'true') {
```

Comment:
You could name this `MAINTENANCE_MODE` but I'm not too precious 😄

Reply:
And I would add this (as `false`) to the `env` section of
- `helm_deploy/values-dev.yaml`
- `helm_deploy/values-preprod.yaml`
- `helm_deploy/values-prod.yaml`

Reply:
Updated all these changes